integration/nwo/token/platform.go

```diff
@@ -224,6 +226,19 @@ func (p *Platform) SetPublicParamsGenerator(name string, gen generators.PublicPa
 	p.PublicParamsGenerators[name] = gen
 }
 
+func (p *Platform) UpdatePublicParams(tms *topology2.TMS, publicParams []byte, chaincodeId string, chaincodeVersion string) {
```

Comment:
would it be possible to make this agnostic to the backend?
This assumes that the TMS is for Fabric.
Maybe we can make it generic and the network handler take care of the other details.
What do you think>